src/main/java/frc/robot/config/FluidConstant.java

```diff
@@ -76,14 +76,6 @@ public T getValue() {
         return this.value;
     }
     
-    /**
-     * Get method for implementing the Supplier interface
-     * @return The current value of this constant
-     */
-    public T get() {
-        return this.value;

```

Comment:
I notice a lot of people who have this deletion in their code reviews. Any idea what's going on?